packages/python-sat/patches/force_malloc.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/cardenc/pycard.cc b/cardenc/pycard.cc
-index 4ae27fe..5941d0a 100644
+index c4d2ead..a0a30db 100644
```

Comment:
Did anything change in this patch? It looks like only whitespace changes? Did the patch not apply cleanly without changing the whitespace?

Reply:
Actually, nothing changed in this patch. I guess you are right that it should not be updated then.

Reply:
Ah, now I remember why I updated this patch too. The reason is that line numbers in the affected files changed and I thought it would be safer to update the patch accordingly.